src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/IndexRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class IndexRequest
+    {
+        public string HashedAccountId { get; set; }
+        public int PageNumber { get; set; }
```

Comment:
May not be the place for it, but if we don't set default PageNumber = 1, on initially loading the page the 12 month download restriction is going to apply to the data that is shown

Reply:
yes, it was there before ... wonder why it's removed.